patch_rebaser/patch_rebaser.py

```diff
@@ -119,6 +234,7 @@ def main():
 
     # The next variables come from patch_rebaser.ini
     rebaser_config = get_rebaser_config()
+    dev_mode = rebaser_config.getboolean('DEFAULT', 'dev_mode')
```

Comment:
I wonder if we should start adding default options for new options in the ini file. What do you think?

Reply:
Yes, it's something I want to do as well. The "fallback" option for configparser appears to only be available with Python 3 so I haven't done it yet, but the custom function we'd need to do that shouldn't be complicated at all. Though I would prefer to do this in a separate PR, if that sounds reasonable.

Reply:
Sure, that sounds good to me.